filebeat/processor/add_kubernetes_metadata/matchers.go

```diff
@@ -86,7 +86,7 @@ func (f *LogPathMatcher) MetadataIndex(event common.MapStr) string {
 		logp.Debug("kubernetes", "Incoming log.file.path value: %s", source)
 
 		if !strings.Contains(source, f.LogsPath) {
-			logp.Debug("kubernetes", "Error extracting container id - source value does not contain matcher's logs_path '%s'.", f.LogsPath)
+			logp.Err("Error extracting container id - source value does not contain matcher's logs_path '%s'.", f.LogsPath)
```

Comment:
We are in process of refactoring log related like `logp.Err` in https://github.com/elastic/beats/issues/15699. I can create a separate PR for this after this PR gets merged.

Reply:
thank you! 😊